app/views/manifests/show.html.erb

```diff
@@ -0,0 +1,89 @@
+<div class="usa-width-two-thirds">
+  <h2>
+    Manifest <%= @manifest.tracking_number %></span>
+  </h2>
+
+  <h3>Generator</h3>
+  <dl>
+    <dt>Generator ID Number</dt>
+    <dd><%= @manifest.tracking_number %></dd>
+    <dt>Emergency Phone Number</dt>
+    <dd><%= @manifest.tracking_number %></dd>
+    <dt>Manifest Tracking Number</dt>
+    <dd><%= @manifest.tracking_number %></dd>
+    <dt>Generator's Name</dt>
+    <dd><%= @manifest.generator_name %></dd>
+    <dt>Mailing Address</dt>
+    <dd><%= @manifest.tracking_number %></dd>
+    <dt>Phone</dt>
+    <dd><%= @manifest.tracking_number %></dd>
+  </dl>
+
+  <h3>Transporter</h3>
```

Comment:
there could be any number of transporters. not sure if you'd rather deal with it now or later, but we should make sure this layout deals with that nicely